### PR TITLE
Convert distance to deg

### PR DESCRIPTION
**Pull request by calum-chamberlain**
Previously event.origins[x].arrivals[y].distance was incorrectly not converted from km to degrees, this commit fixes that.

### CODE REVIEW

**Comment by QuLogic** [comment]
I don't know what's up with the build at the moment, but I'd have thought this might break some test, no? Otherwise, maybe a test should be added.

**Comment by calum-chamberlain** [comment]
You are almost certainly right @QuLogic possibly premature PR - need to convert the other way too, will look again tomorrow and run some tests.

**Comment by calum-chamberlain** [comment]
So I added the reverse option and tested - it did fail the previous tests, but I amended the test.  I can't immediately think of a good way to test whether the units are correct when read in/written out - any ideas?

**Comment by krischer** [comment]
Can you not just manually calculate the distances for a couple of tests files and check if they are in the correct ballpark?

**Comment by calum-chamberlain** [comment]
I think this is done, not sure what the issues with appveyor are, they don't appear to be related to this?

**Comment by QuLogic** [comment]
The former is unrelated, the latter is fixed by #1614, I think.

File: obspy/io/nordic/tests/test_nordic.py [hunk 1]
```diff
@@ -456,8 +456,24 @@ def test_read_picks_across_day_end(self):
             self.assertTrue(pick.time in pick_times)
         self.assertEqual(event_2.origins[0].time, event.origins[0].time)
 
-
-def test_similarity(event_1, event_2):
+    def test_distance_conversion(self):
+        """Check that distances are converted properly."""
```
Comment (krischer) [inline obspy/io/nordic/tests/test_nordic.py]: Can you please use 
```
def ...():
    """
    TEXT
    """
    ...
```

as a docstring style to be consistent with the rest of ObsPy.

File: obspy/io/nordic/tests/test_nordic.py [hunk 2]
```diff
@@ -456,8 +456,24 @@ def test_read_picks_across_day_end(self):
...
+        testing_path = os.path.join(self.testing_path, '01-0411-15L.S201309')
+        cat = read_events(testing_path)
+        event = cat[0]
+        self.assertEqual(sorted(event.origins[0].arrivals,
+                                key=lambda x: x.distance)[0].distance,
+                         0.035972864236749225)
```
Comment (krischer) [inline obspy/io/nordic/tests/test_nordic.py]: This should probably be an "assertAlmostEqual" just to deal with floating point inaccuracies. Tests seem to pass fine but it cannot hurt to be safe.

**Comment by krischer** [comment]
Thanks for this!

**Pull request merged**